src/lib/validation/validationRule.ts

```diff
@@ -6,7 +6,7 @@ export const validationRule = (
   validator: async (_: RuleObject, value: string) => {
     try {
       const reference = value || '';
```

Comment:
could this be removed? "|| '"" ??

Reply:
Even though ant assures that it is "string", some times it can be undefined. Thus reference.length would crash the program. This ensures that it **always** is a string.